app.css

```diff
@@ -1,8 +1,7 @@
 
 #frame {
     display: inline-block; 
-    width: 720px; 
-    height: 405px; 
     background-image: url('videos/test_vid/0/0/0.jpg');
+    transition: background 1s linear;
```

Comment:
Removing the width and height fixed weird scaling. The transition is to try to remove the white flickering in between frame transitions.